activerecord/lib/active_record/tasks/database_tasks.rb

```diff
@@ -494,9 +522,9 @@ def cache_dump_filename(db_config_name, schema_cache_path: nil)
         schema_cache_path || ENV["SCHEMA_CACHE"] || File.join(ActiveRecord::Tasks::DatabaseTasks.db_dir, filename)
       end
 
-      def load_schema_current(format = ActiveRecord.schema_format, file = nil, environment = env)
+      def load_schema_current(file = nil, environment = env)
```

Comment:
We need to deprecate the `format` options that was remove here.

Reply:
Thank you so much for your feedback!  I apologize for the formatting errors.

I'm about to commit one idea for the deprecation here.  Let me know if you'd like me to do it differently.  Thanks again!